fendermint/vm/interpreter/src/fvm/checkpoint.rs

```diff
@@ -331,7 +331,7 @@ pub fn emit_trace_if_check_checkpoint_finalized<DB>(
 where
     DB: Blockstore + Clone,
 {
-    if !gateway.enabled(state)? {
+    if !gateway.is_root(state)? {
```

Comment:
(I know this method already existed, but it didn't appear to be called before, so taking a few moments to revisit the naming.) 

I think we should rename this to `is_anchored`, since it's only gonna be called inside subnets (IPC is a client for subnets). We tend to use the term _root_ to refer to existing L1s where we root a hierarchy.